app/views/imports/_form_status.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="card shadow border-0">
+  <div class="card-header-success">
+    <h5 class="card-title">Import</h5>
+  </div>
+
+  <div class="card-body row">
+    <div class="col-12 mb-4 prx-field-group">
+      <div class="form-floating">
+        <%= form.select :feed_source, ["RSS"] %>
+        <%= form.label :feed_source, "Feed source" %>
```

Comment:
isn't functional yet, but here to hold space for what will eventually be added as a source selector.